Tryouts/Core/BasicModels/README.md

```diff
@@ -0,0 +1,8 @@
+<!-- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. -->
+
+# MorganStanley.ComposeUI.Common.BasicModels
+============================================
+
+BasicModels is a project containing some primitive view model classes with properties and all the functionatily related to them. 
```

Comment:
To me this description doesn't help in understanding what exactly are 'basic models'.

Reply:
Here is my update:

BasicModels is a project containing generic View Models that can be utilized possibly
by several plugins and/or the shell. For example it contains the module displaying functionality and can
also contain generic view models for authentication/authorization and others.